util/network-devp2p/src/node_table.rs

```diff
@@ -27,6 +27,7 @@ use network::{Error, ErrorKind, AllowIP, IpFilter};
 use discovery::{TableUpdates, NodeEntry};
 use ip_utils::*;
 use serde_json;
+use std::io;
```

Comment:
please place imports in order, ideally in line 23, together with other `std` imports

tip: if you are using vim, you can easily sort everything by typing `:sort` when in visual mode :)